docs/user/installation_and_configuration.rst

```diff
@@ -301,6 +295,120 @@ In this case you may download SLF4J from http://www.slf4j.org/download.html. Ext
+
+The instructions below assume that the ``geomesa-kafka-$KAFKAVERSION_2.11-$VERSION`` directory is kept in the
+``geomesa-kafka/geomesa-kafka-dist/geomesa-kafka-$KAFKAVERSION-dist/target/`` directory, but the tools distribution may be moved elsewhere
+as desired.
+
+.. note::
+
+    You can configure environment variables and classpath settings in geomesa-kafka-$KAFKAVERSION_2.11-$VERSION/bin/geomesa-env.sh.
+
+In the ``geomesa-kafka-$KAFKAVERSION_2.11-$VERSION`` directory, run ``bin/geomesa configure`` to set up the tools.
+
+.. code-block:: bash
+
+    ### in geomesa-kafka-$KAFKAVERSION_2.11-$VERSION:
+    $ bin/geomesa-kafka configur
+    Using GEOMESA_KAFKA_HOME as set: /path/to/geomesa-kafka-$KAFKAVERSION_2.11-$VERSION
+    Is this intentional? Y\n y
+    Warning: GEOMESA_LIB already set, probably by a prior configuration.
+    Current value is /path/to/geomesa-kafka-$KAFKAVERSION_2.11-$VERSION/lib.
+
+    Is this intentional? Y\n y
+
+    To persist the configuration please update your bashrc file to include:
+    export GEOMESA_KAFKA_HOME=/path/to/geomesa-kafka-$KAFKAVERSION_2.11-$VERSION
+    export PATH=${GEOMESA_KAFKA_HOME}/bin:$PATH
+
+Update and re-source your ``~/.bashrc`` file to include the ``$GEOMESA_KAFKA_HOME`` and ``$PATH`` updates.
+
+.. warning::
+
+    Please note that the ``$GEOMESA_KAFKA_HOME`` variable points to the location of the ``geomesa-kafka-$KAFKAVERSION_2.11-$VERSION``
+    directory, not the main geomesa binary distribution directory!
+
+.. note::
+
+    ``geomesa-kafka`` will read the ``$GEOMESA_EXTRA_CLASSPATHS`` environment variable to load any
+    additional jars into the classpath. Use the ``geomesa classpath`` command in order to see what
+    JARs are being used.
+
+Due to licensing restrictions, dependencies for shape file support and raster
+ingest must be separately installed. Do this with the following commands:
+
+.. code-block:: bash
+
+    $ bin/install-jai.sh
+    $ bin/install-jline.sh
+
+Test the command that invokes the GeoMesa Tools:
+
+.. code-block:: bash
+
+    $ geomesa-kafka
+    Using GEOMESA_KAFKA_HOME = /path/to/geomesa-kafka-$KAFKAVERSION_2.11-$VERSION
+    Usage: geomesa [command] [command options]
+      Commands:
+        create           Create a feature definition in a GeoMesa catalog
+        deletecatalog    Delete a GeoMesa catalog completely (and all features in it)
+        deleteraster     Delete a GeoMesa Raster Table
+        describe         Describe the attributes of a given feature in GeoMesa
+        env              Examine the current GeoMesa environment
+        explain          Explain how a GeoMesa query will be executed
+        export           Export a GeoMesa feature
+        getsft           Get the SimpleFeatureType of a feature
+        help             Show help
+        ingest           Ingest a file of various formats into GeoMesa
+        ingestraster     Ingest a raster file or raster files in a directory into GeoMesa
```

Comment:
these aren't all available on kafka tools - either the distro is messed up or it was copied wrong.